c31-swynlake/reactive-nutshell
# Brainbiranbrian

# Description

Added in function to retrieve active users friends, and store them in state for other components to use. 

Applies to issue # 8

## Type of change

Please delete options that are not relevant.



# Testing Instructions

1. git fetch checkout brainbiranbrian
1. login and check that state in application views is returning the ID of active user's friends.



# Checklist:
